docs/resources/User.md

```diff
@@ -33,6 +33,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | bot?          | boolean   | whether the user belongs to an OAuth2 application                                                    | identify              |
 | system?       | boolean   | whether the user is an Official Discord System user (part of the urgent message system)              | identify              |
 | mfa_enabled?  | boolean   | whether the user has two factor enabled on their account                                             | identify              |
+| banner?       | ?string   | the user's banner, or null if unset                                                                  | identify              |
+| banner_color? | ?number   | the user's banner color as a hex color                                                               | identify              |
```

Comment:
This should be decimal, not hex. A decimal of a hex value to be more precise.

Reply:
Feels kind of weird to me to refer to it as decimal given it's not really decimal? Embeds just document `color` as "color code of the embed" so something like "color of the user's banner" might work better.

Reply:
[role](https://discord.com/developers/docs/topics/permissions#role-object-role-structure ) color has a longer description ~~because obviously it can't be consistent~~